air/src/options.rs

```diff
@@ -68,7 +68,7 @@ pub enum FieldExtension {
 ///    time decrease the number of queries in such a way that the proofs become smaller.
 /// 4. Grinding factor - higher values increase proof soundness, but also may increase proof
 ///    generation time. More precisely, conjectured proof soundness is bounded by
-///    `num_queries * log2(blowup_factor) + grinding_factor`.
+///    `num_queries * blowup_factor.ilog2() + grinding_factor`.
```

Comment:
Here, and in other places in this file, I'd probably keep the comments as they were as they more for explaining concepts rather than for demonstrating code.